### ISSUE TITLE
[request] setting not to set up chrony

### ISSUE THREAD

**Issue by ozota1** [NVIDIA/deepops#593]
deepops 20.06 installs and setups chrony .
But prefferd NTP solution depends on user's environment.
And if there is existing configuration of chrony, the configuration will be overrided and it might be needed to be repaired. 

So I would like a setting not to set up chrony.

for example, by the below setting in group_vars/all.yml, deepops will do nothing about chrony.
```
chrony_setup: "no"
# if chrony_setup is "no", the below settings can be omitted
chrony_service_state: "started"
chrony_service_enabled: "yes"
chrony_timezone: "Etc/UTC"
chrony_config_server:
  - 0.pool.ntp.org
  - 1.pool.ntp.org
  - 2.pool.ntp.org
  - 3.pool.ntp.org
```

**Comment by ajdecon** [comment]
Sure, this is a good idea. I'll take a look at adding this.

**Comment by ajdecon** [comment]
Resolved by #594

**Comment by ozota1** [comment]
Thank you!
